BoardView.cpp

```diff
@@ -334,38 +334,58 @@ void BoardView::HandleInput() {
 	}
 	if (!io.WantCaptureKeyboard) {
 
-		// (I/+) and (O/-) keys as alternate zoom. Will zoom on the centre of the viewport, not on the mouse.
-		if (ImGui::IsKeyPressed('I') || ImGui::IsKeyPressed(187)) {
+		//Ctrl-Q as alternative quit shortcut (Alt-F4 is already handled by Windows)
+		if (ImGui::IsKeyPressed('Q') && ImGui::IsKeyDown(17)) {
+			m_wantsQuit = true;
+		}
+
+		// (I/+/PgUp) and (O/-/PgDn) keys as alternate zoom. Will zoom on the centre of the viewport, not on the mouse.
+		
+		//TODO numpad +/- scancodes
+		if ( ImGui::IsKeyPressed('I')
+			|| ImGui::IsKeyPressed(187)
```

Comment:
If possible, could you replace all hard coded numbers with `GetKeyIndex`?  That should aid readability (and porting to GLFW later)